client/ayon_core/vendor/python/scriptsmenu/action.py

```diff
@@ -112,20 +112,21 @@ def run_command(self):
         Run the command of the instance or copy the command to the active shelf
         based on the current modifiers.
 
-        If callbacks have been registered with fouind modifier integer the
+        If callbacks have been registered with found modifier integer the
         function will trigger all callbacks. When a callback function returns a
         non zero integer it will not execute the action's command
-
         """
 
         # get the current application and its linked keyboard modifiers
         app = QtWidgets.QApplication.instance()
         modifiers = app.keyboardModifiers()
+        if not QT6:
+            modifiers = int(modifiers)
```

Comment:
We could use the advantage that modifiers in PySide6 are enum items (they have `name` and `value` attributes).
```suggestion
        if not hasattr(modifiers, "value"):
            modifiers = int(modifiers)
```

Reply:
Feels a bit less explicit than it being Qt6 related - why do you feel that's nicer?

Reply:
I didn't say nicer, but it does not require import of `QT6`.